src/store/store.js

```diff
@@ -1,11 +1,12 @@
 // @flow
```

Comment:
It might be better, I have been trying to move away from files named `index.js`, the only drawback is that we have to repeat the name when importing them (`import store from 'store/store'`), but the advantage is that it's easier to search for a given file with the quick lookup, and we avoid situations like these:

![image](https://user-images.githubusercontent.com/1882507/46481280-0c133100-c7c1-11e8-9ed4-2e31f851d059.png)

![image](https://user-images.githubusercontent.com/1882507/46481181-d2422a80-c7c0-11e8-8097-de18644b5138.png)

What do you think about this?

Reply:
I see your point. It is kind of annoying to have a bunch of files with same name in different directories. I am leaving you to make the best decision. :-p